# Generated by Django 4.1.4 on 2023-03-01 18:37

from django.db import migrations, models
import users.models


class Migration(migrations.Migration):

    dependencies = [
        ('users', '0004_rename_accepted_friends_is_accepted'),
    ]

    operations = [
        migrations.AlterField(
            model_name='message',
            name='data_created',
            field=models.DateTimeField(auto_now_add=True),
        ),
        migrations.AlterField(
            model_name='user',
            name='username',
            field=models.CharField(error_messages={'unique': 'A user with that username already exists.'}, help_text='Required. 150 characters or fewer. Letters, digits and @/./+/-/_ only.', max_length=150, unique=True, validators=[users.models.contain_invalid_symbols_in_username], verbose_name='username'),
        ),
    ]
